neqo-client/src/main.rs

```diff
@@ -519,6 +526,7 @@ fn main() -> Res<()> {
                     remote_addr,
                     &format!("{}", host),
                     &[url],
+                    ciphers,
```

Comment:
The API on this is starting to get uncomfortably wide.  I wonder if we need to start breaking this down more.

Reply:
Maybe. For now I've added allow(clippy::too_many_arguments) to make clippy happy.